### ISSUE TITLE
Mobile VO bullets no longer read out after changes are made

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab-basics#310]
For https://github.com/phetsims/qa/issues/702. Mobile VO on iPadOS 14. May be related to other Mobile VO issues that have been showing up.
When reading through the scene description, the bullet points describing the scene seem to disappear if you have made changes.
Steps:
1. Swipe two fingers up to read through the scene description
2. Adjust the positions and masses of the objects
3. Swipe up again to read through the scene description with missing info.

https://user-images.githubusercontent.com/41024075/132735176-1ede1eb7-9a69-4b1e-8a5b-4b9ef43428e8.MOV

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe - seems really similar to the issue in john-travoltage: https://github.com/phetsims/john-travoltage/issues/449

**Comment by KatieWoe** [comment]
This does seem to be fixed in rc.3 when looking at iPadOS 15

**Comment by jessegreenberg** [comment]
OK thanks for verifying @KatieWoe. Ill close this issue but I imagine we may run into this in other sims still since it is still a problem for john-travoltage. Ill make a general issue in a11y-research and probably submit an apple bug report about this.
